test/karma.conf.js

```diff
@@ -97,7 +97,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['Chrome'],
```

Comment:
```suggestion
    browsers: [process.env.TRAVIS ? 'Chrome_travis_ci' : 'Chrome'],
    
    customLaunchers: {
      Chrome_travis_ci: {
        base: 'Chrome',
        flags: ['--no-sandbox']
      }
    },
```

Reply:
Oh wow, thank you for checking this out! I am still getting this error however: https://github.com/publiclab/Leaflet.DistortableImage/pull/685#issuecomment-656410148 (`Cannot start Chrome`). I'll need to do some more research into it to unravel that a bit. Thanks!

Reply:
@MLefebvreICO im working on this issue in #716 and I managed to get the chrome launcher working in travis without no sandbox, but failing to do the same for gitpod. Considering just going with --no-sandbox, but I am seeing online that it's not good practice security-wise. Not really educated on the matter, are there certain scenarios where it is ok to do anyway? would love your input!

Reply:
@sashadev-sky I think it might be back in the days (2017) where we had no choice but to use it in some tests env or the browser wouldn't start properly. Thinking about it, it may have been fixed nowadays.

As for your gitpod issue, I'm gonna comment on #716 for that.

Reply:
thanks for helping out ! @MLefebvreICO